assets/css/main.css

```diff
@@ -0,0 +1,23 @@
+.md-toolbar-tools {
+  background: #FFEB00 !important;
+  color: #000; }
+
+.md-grid-tile-footer {
+  background: #FFEB00 !important;
+  color: #000; }
+
+.md-card-content {
+  background: #FFEB00 !important;
+  color: #000; }
+
+.md-subheader-content {
+  color: #000 !important;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/triplez/open-event-webapp/file/2406579142/issues/source?bid=3233857&fileBranchId=3233877#l14)